Dockerfile.dapper

```diff
@@ -16,9 +17,14 @@ ENV KUSTOMIZE_VERSION v3.10.0
 # kontainer-driver-metadata branch to be set for specific branch other than dev/master, logic at rancher/rancher/pkg/settings/setting.go
 ENV CATTLE_KDM_BRANCH=dev-v2.6
 
-RUN apt-get update && \
-    apt-get install -y gcc ca-certificates git wget curl vim less file xz-utils unzip iproute2 && \
-    rm -f /bin/sh && ln -s /bin/bash /bin/sh
+RUN if [ -x /usr/bin/zypper ]; then \
+        zypper ar --refresh --priority 100 'https://updates.suse.com/SUSE/Products/SLE-BCI/$releasever_major-SP$releasever_minor/$basearch/product/' SLE_BCI && \
```

Comment:
Variables `$releasever_major`, `$releasever_minor` and `$basearch` probably should be defined.

Reply:
No, those variable are internal to zypper (see man zypper) and resolved when zypper is run. This allows the repository url to be version and arch agnostic.

Reply:
@fcrozat thank you, this is useful! This thread can be marked as resolved.

Reply:
Last comment in this particular topic: the zypper ar call will be removed once SLE Base Image is updated to have SLE_BCI pre-configured.